tests/test_platform_factory.py

```diff
@@ -51,11 +53,11 @@ def _load_platform_tags_from_file(filename: Union[Path, str]) -> list[str]:
         return data["python_tags"]
 
     @classmethod
-    def _testdata(cls, path=".") -> Iterable[RegressionTestData]:
+    def _testdata(cls) -> Iterable[RegressionTestData]:
         """Yields testdata for subtests."""
         platforminfo_files = [
             entry
-            for entry in Path(path).iterdir()
+            for entry in importlib.resources.files(tests).iterdir()
```

Comment:
You should use pyenv to install py37 and then run `poetry env use 3.7` to use it. This will ensure that you don't use features that aren't available in py37. You want req2flatpak to be usable with py37 right?

Reply:
> You want req2flatpak to be usable with py37 right?
yes, that's correct.

> You should use pyenv to install py37 and then run poetry env use 3.7 to use it. 
you are right, thanks for pointing out how to do this. (I still need to set this up on my machine).

Reply:
btw, do you have a recommendation for how to install pyenv on ubuntu? (no official package available?)

Reply:
> btw, do you have a recommendation for how to install pyenv on ubuntu? (no official package available?)

I think I used [pyenv-installer](https://github.com/pyenv/pyenv-installer).

Reply:
thanks, I got pyenv working now. I'll open another branch to make the tests compatible again with py3.7